app/views/take_whole_pot_calculator/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>Taking your whole pot in one go</h1>
+
+<div class="calculator">
+  <h2>Estimate how much you would get</h2>
+
+  <form action="/take-whole-pot/results" method="post">
```

Comment:
I'm not sure it makes sense to use `POST` given we're not creating a resource.

If we used `GET` we're saying "give me this resource, here's some additional context in the form of params".